libs/explored-types/src/types.ts

```diff
@@ -174,6 +173,17 @@ export type ExplorerFileContract = FileContractElement & {
 
   proofIndex: ChainIndex | null
   proofTransactionID: TransactionID | null
+
+  id: FileContractID
+  filesize: number
+  fileMerkleRoot: Hash256
+  windowStart: number
+  windowEnd: number
+  payout: Currency
+  validProofOutputs: (SiacoinOutput & { id: SiacoinOutputID })[]
```

Comment:
This type now has no lineal relationship to core `SiacoinElement`, and I'm still deciding the best way to handle that. You're right in that it did, at first. `explored` types have changed quite a bit since this library was first written. These are likely "final" outcomes for this typing:

1. One type of `value` and `address` and a wrapping type that adds `id`.
2. One type of those three values. 

But note: nothing more, in terms of properties--at least right now. 

What we have above is an intermediate solution that meets the changes made to `explored` only insofar as they serve the explorer contracts route API change. There's several more changes that need to be made in this file but none other that affect explorer, to my knowledge. It's a separation of concerns, was my thinking.